src/components/Navigator/NavigatorCard.vue

```diff
@@ -28,6 +33,18 @@
         @keydown.up.exact.capture.prevent="focusPrev"
         @keydown.down.exact.capture.prevent="focusNext"
       >
+        <Reference
+          v-if="technology"
+          :id="INDEX_ROOT_KEY"
+          :url="technologyPath"
+          class="technology-title"
+          @click.alt.native.prevent="toggleAllNodes"
```

Comment:
This is cool but hard to discover. 
I also noticed that the technology name is indented the same as its children. Could we decrease its left margin and create some visual separation? Maybe in that world we can have another tooltip to toggle all nodes within the technology.

Reply:
I agree that it's hard to discover but deleting it would mean a regression from the current implementation. Opening an issue to re-think this functionality and UI would be the best idea to proceed.